open-media-match/src/OpenMediaMatch/database.py

```diff
@@ -64,8 +69,36 @@ def as_storage_iface_cls(self) -> BankContentConfig:
         )
 
 
+@dataclass
```

Comment:
This interacts strangely with db.Model and the columns - I found a few edge cases during working that I couldn't find a workaround to. I'm not sure its worth keeping :/